nanostack-event-loop/platform/eventloop_config.h

```diff
@@ -19,8 +19,8 @@
 /*
  * Undefine all internal flags before evaluating the configuration
  */
-#undef NS_EVENTLOOP_USE_TICK_TIMER
-#undef NS_EXCLUDE_HIGHRES_TIMER
+//#undef NS_EVENTLOOP_USE_TICK_TIMER
```

Comment:
Yes, these shouldn't have been undeffing. Suggest leaving them as comment out `#define`s, and replace the "undefine" comment with a brief explanation of each option.

Maybe

TICK_TIMER: "Define to use platform_tick_timer API for low-resolution timers"
EXCLUDE_HIGH_RES_TIMER: "Define if platform does not provide high-resolution platform_timer API"